### PR TITLE
[webpack] fix WebpackConfig.Node value type

### PR DESCRIPTION
**Pull request by ZSkycat**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://webpack.js.org/configuration/node/

### CODE REVIEW

**Review by r3nya — APPROVED** [review]

**Comment by ZSkycat** [comment]
sorry, I accidentally deleted the branch. what should I do?

File: types/webpack/index.d.ts [hunk 1]
```diff
@@ -349,14 +350,16 @@ declare namespace webpack {
     type ExternalsFunctionElement = (context: any, request: any, callback: (error: any, result: any) => void) => any;
 
     interface Node {
-        console?: boolean;
+        console?: boolean | 'mock';
```
Comment (mohsen1) [inline types/webpack/index.d.ts]: why this and `process` can't be `"empty"`?

Comment (r3nya) [reply types/webpack/index.d.ts]: https://webpack.js.org/configuration/node/#node-console
https://webpack.js.org/configuration/node/#node-process
🤔

Comment (ZSkycat) [reply types/webpack/index.d.ts]: see https://webpack.js.org/configuration/node/#node-process

try it
```
node: {
        process: 'empty'
}

npm run build
```

Output result
```
> webpack --config webpack.development.js

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.node should be one of these:
   false | object { Buffer?, __dirname?, __filename?, console?, global?, process?, … }
   -> Include polyfills or mocks for various node stuff.
   Details:
    * configuration.node should be false
    * configuration.node.process should be one of these:
      false | true | "mock"
      -> Include a polyfill for the 'process' variable
```

**Review by alan-agius4 — APPROVED** [review]

**Pull request merged**